package.json

```diff
@@ -3,46 +3,69 @@
   "version": "0.3.0",
   "description": "A frontend for IPFS",
   "dependencies": {
-    "bootstrap": "3.3.5",
-    "d3": "3.5.6",
-    "debug": "2.2.0",
-    "font-awesome": "4.5.0",
-    "i18next-client": "1.11.1",
-    "ipfs-api": "2.7.0",
-    "ipfs-geoip": "1.4.5",
-    "jquery": "2.1.4",
-    "lodash": "3.10.1",
-    "react": "0.14.2",
-    "react-bootstrap": "0.27.3",
-    "react-dom": "0.14.2",
-    "react-localstorage": "0.2.8",
-    "react-router": "0.13.5",
-    "three": "0.73.0"
+    "bootstrap": "^3.3.5",
+    "d3": "^3.5.6",
+    "debug": "^2.2.0",
+    "font-awesome": "^4.5.0",
+    "i18next": "^2.0.5",
+    "i18next-browser-languagedetector": "0.0.11",
+    "i18next-localstorage-cache": "0.0.4",
+    "i18next-sprintf-postprocessor": "0.0.6",
+    "i18next-xhr-backend": "0.0.5",
+    "ipfs-api": "^2.10.0",
+    "jquery": "^2.1.4",
+    "lodash": "^3.10.1",
+    "react": "^0.14.2",
+    "react-bootstrap": "^0.28.1",
+    "react-dom": "^0.14.2",
+    "react-localstorage": "^0.2.8",
+    "react-router": "^0.13.5",
+    "three": "^0.73.0"
   },
   "devDependencies": {
-    "babel": "^5.8.29",
-    "babel-core": "^5.8.33",
-    "babelify": "^5.0.5",
-    "brfs": "^1.4.1",
-    "browserify": "^12.0.1",
-    "bygg": "^1.1.0",
-    "bygg-plugins-light": "1.0.3",
+    "autoprefixer": "^6.2.1",
+    "babel": "^6.3.13",
+    "babel-core": "^6.3.21",
+    "babel-eslint": "^5.0.0-beta6",
+    "babel-loader": "^6.2.0",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-react": "^6.3.13",
+    "babel-preset-stage-0": "^6.3.13",
+    "chai": "^3.4.1",
+    "css-loader": "^0.23.0",
+    "eslint": "^1.10.3",
+    "eslint-config-standard": "^4.4.0",
+    "eslint-config-standard-react": "^1.2.1",
+    "eslint-plugin-react": "^3.12.0",
+    "eslint-plugin-standard": "^1.3.1",
+    "file-loader": "^0.8.5",
+    "hjs-webpack": "^6.0.0",
+    "https-browserify": "0.0.1",
+    "ipfs-api": "^2.10.1",
+    "json-loader": "^0.5.4",
+    "karma": "^0.13.15",
+    "karma-chrome-launcher": "^0.2.2",
+    "karma-firefox-launcher": "^0.1.7",
+    "karma-mocha": "^0.2.1",
+    "karma-sourcemap-loader": "^0.3.6",
+    "karma-webpack": "^1.7.0",
     "less": "^2.5.3",
+    "less-loader": "^2.2.2",
+    "mocha": "^2.3.4",
+    "postcss-loader": "^0.8.0",
     "pre-commit": "^1.1.2",
-    "proxy-middleware": "^0.15.0",
-    "standard": "5.3.1"
+    "react-addons-test-utils": "^0.14.3",
+    "stream-http": "^2.0.2",
+    "style-loader": "^0.13.0",
+    "url-loader": "^0.5.7",
+    "webpack": "^1.12.9",
+    "webpack-dev-server": "^1.14.0"
```

Comment:
I'm actually considering doing sth like that, gets painful doing the same things everywhere over and over again..